src/ducks/transactions/actions/ReimbursementStatusAction/ReimbursementStatusAction.jsx

```diff
@@ -113,7 +122,13 @@ export class DumbReimbursementStatusAction extends React.PureComponent {
 
 const ReimbursementStatusAction = compose(
   translate(),
-  withMutations()
+  withMutations(),
+  connect(state => {
+    const settings = getSettings(state)
+    return {
+      healthReimbursementLateLimit: getHealthReimbursementLateLimit(settings)
```

Comment:
Why don't you use the selector `getHealthReimbursementLateLimitSelector` here?

Reply:
Good catch, fixed